bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/handler/YamahaZoneThingHandler.java

```diff
@@ -287,11 +287,11 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                     zoneControl.setVolumeDB(((DecimalType) command).floatValue());
                     break;
                 case CHANNEL_VOLUME:
-                    if (command instanceof DecimalType) {
-                        zoneControl.setVolume(((DecimalType) command).floatValue());
-                    } else if (command instanceof IncreaseDecreaseType) {
+                    if (command instanceof DecimalType decimalType) {
```

Comment:
For consistency, perhaps you could use the same name here and below, i.e. either `decimalType` or `commandAsDecimalType`. Or even `decimalCommand`.

Reply:
Done.